apps/coinsence-kit/truffle.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * https://github.com/aragon/aragonOS/blob/v4.0.0/truffle-config.js
```

Comment:
we should use a shared truffle.js; something like: 
```js
module.exports = require("../../truffle.js");
```
but can do that in a different cleanup PR

Reply:
okay